src/components/nodes/adm/AdmNodesTable.vue

```diff
@@ -29,23 +29,33 @@ export default defineComponent({
   },
   setup() {
     const store = useStore()
+
     const admNodes = computed(() => {
       const arr = store.getters['nodes/adm']
 
       return [...arr].sort(sortNodesFn)
     })
 
+    const isAllChecked = ref(false)
+
+    const isAllNodesEnabled = !admNodes.value.some(node => node.active === false)
+    if (isAllNodesEnabled) isAllChecked.value = true
```

Comment:
You don't need to define a new state (line 39), since the value can be computed from the actual Vuex state

```suggestion
    const isAllChecked = computed(() => admNodes.value.every(node => node.active))
```

Reply:
I defined not a new state, but a variable used in v-model, see NodesTableHead. That additional check is made for the case of the new tab switch with all nodes have been already checked before.